app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_return_zero_if_age_less_than_15() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_return_1_if_age_less_than_24() -> None:
+    assert get_human_age(23, 23) == [1, 1]
+
+
+def test_return_2_if_age_less_than_28() -> None:
+    assert get_human_age(27, 27) == [2, 2]
+
+
+def test_return_different_values_if_age_equals_28_and_more() -> None:
+    assert get_human_age(28, 28) == [3, 2]
```

Comment:
The tests currently do not cover edge cases such as negative numbers, zero, or very large numbers. Consider adding tests for these scenarios to ensure robustness, as per the checklist requirement.